README.md

```
@@ -1,184 +1,110 @@
-```bash
-supabase bootstrap
+# Build for production
+npm run build
 ```
 
-Or using npx:
+## Project Structure
 
-```bash
-npx supabase bootstrap
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)